docs/dev/logging.adoc

```diff
@@ -6,7 +6,9 @@ https://godoc.org/github.com/golang/glog[Glog] is used for V style logging in `o
 
 Every `odo` command takes an optional flag `-v` that must be used with an integer log level in the range from 0-9. Any INFO severity log statement that is logged at a level lesser than or equal to the one passed with the command alongside `-v` flag will be logged to STDOUT.
 
-All ERROR severity level log messages will always be logged regardless of the passed `v` level.
+Alternatively, enviroment variable `ODO_LOG_LEVEL` can be used to set the verbosity of the log level in the integer range 0-9. The value set by `ODO_LOG_LEVEL` can be overridden by explicitly passing the `-v` command line flag to the `odo` command, in such an event `-v` flag takes precedence over the enviroment variable `ODO_LOG_LEVEL`.
```

Comment:
Changing `command line flag to the odo command, in such an event` to `
command line flag to the odo command. In such an event` makes more sense to me.